github/multiformats.yml

```diff
@@ -124,7 +124,6 @@ repositories:
     archived: false
     collaborators:
       admin:
-        - andyschwab
```

Comment:
I assume this is OK?

Reply:
It's for this repo - https://github.com/multiformats/.github. By the looks of it, we could just archive that repository altogether.

Reply:
yeah, looks like we're not doing anything useful with that, let's archive. Should I just do that or do you want to handle it through the tooling?

Reply:
On it!